video/out/wayland_common.h

```diff
@@ -151,6 +151,11 @@ struct vo_wayland_state {
     uint32_t keyboard_code;
     int mpkey;
     int mpmod;
+    double axis_value_vertical;
+    int32_t axis_value120_vertical;
+    double axis_value_horizontal;
+    int32_t axis_value120_horizontal;
+    bool axis_is_wheel;
```

Comment:
```suggestion
    bool axis_is_hires_scroll;
```

Just a naming nit to be clearer.

Reply:
How about `axis_value120_scroll`?